R/teal_slice.R

```diff
@@ -2,6 +2,35 @@
 #'
 #' Create a `teal_slice` object that holds complete information on filtering one variable.
 #'
+# Custom @usage section is needed to that utility functions are listed in the usage and they do not have index entries.
+#' @usage
+#' teal_slice(
+#'   dataname,
+#'   varname,
+#'   id,
+#'   expr,
+#'   choices = NULL,
+#'   selected = NULL,
+#'   keep_na = NULL,
+#'   keep_inf = NULL,
+#'   fixed = FALSE,
+#'   anchored = FALSE,
+#'   multiple = TRUE,
+#'   title = NULL,
+#'   ...
+#' )
+#'
+#' is.teal_slice(x)
+#'
+#' as.teal_slice(x)
+#'
+#' \\method{as.list}{teal_slice}(x, ...)
```

Comment:
I assume this cannot be done with `@method as.list teal_slice` because `@usage` is built manually?